packages/cardano-services/test/cli.test.ts

```diff
@@ -1618,7 +1618,7 @@ describe('CLI', () => {
 
             afterAll(async () => await closeMock());
 
-            it('exposes a HTTP server with healthy state when using CLI options', async () => {
+            it('exposes a HTTP server with healthy state when using CLI options and reaching /get-asset', async () => {
```

Comment:
```suggestion
            it('exposes a HTTP server with healthy state when using CLI options, and /get-asset returns successfully', async () => {
```

Reply:
done in [da0d689](https://github.com/input-output-hk/cardano-js-sdk/pull/691/commits/da0d6895c9995f98aa783ee29399aae26fe11a70)